pkg/services/featuremgmt/registry.go

```diff
@@ -586,5 +586,12 @@ var (
 			Stage:        FeatureStageExperimental,
 			Owner:        grafanaObservabilityTracesAndProfilingSquad,
 		},
+		{
+			Name:         "elasticFiltersToggle",
```

Comment:
It took me a second do understand the right meaning because with "toggle" I thought of feature toggle. WDYT about renaming it to something like `elasticToggleableFilters`?

Reply:
Good idea, done.